resqpy/well/_blocked_well.py

```diff
@@ -1044,15 +1044,13 @@ def import_from_rms_cellio(self, cellio_file, well_name, grid, include_overburde
                                                                      well_name = well_name,
                                                                      grid_name = grid_name)
 
-                self.create_md_datum_and_trajectory(
-                    grid,
-                    trajectory_mds,
-                    trajectory_points,
-                    length_uom,
-                    well_name,
-                    set_depth_zero = True,
-                    set_tangent_vectors = False
-                )  # this is causing issues as the knot_count for the trajectory object is None and so cannot test that knot_count > 1
+                self.create_md_datum_and_trajectory(grid,
+                                                    trajectory_mds,
+                                                    trajectory_points,
+                                                    length_uom,
+                                                    well_name,
+                                                    set_depth_zero = True,
+                                                    set_tangent_vectors = True)
```

Comment:
Please add set_tangent_vectors as an argument to import_from_rms_cellio() and pass the value in from the BlockedWell __init__() method, rather than having it set to a constant value here.